src/macaron/build_spec_generator/common_spec/pypi_spec.py

```diff
@@ -67,15 +67,20 @@ def get_default_build_commands(
 
             match build_tool_name:
                 case "pip":
-                    default_build_commands.append("python -m build".split())
+                    default_build_commands.append("python -m build --wheel -n".split())
                 case "poetry":
-                    default_build_commands.append("poetry build".split())
+                    default_build_commands.append("pip install poetry && poetry build".split())
```

Comment:
What do you think about keeping the default install commands separate from the build commands? For example, we could add a new field to the spec specifically for install commands. I believe this would make the spec easier for downstream users to consume.

Reply:
Yes,  that makes sense.